projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-headline/order-detail-headline.component.html

```diff
@@ -1,18 +1,18 @@
 <ng-container *ngIf="order$ | async as order">
-  <div class="cx-header row">
-    <div class="cx-detail col-sm-12 col-md-4">
+  <div class="cx-header">
```

Comment:
will discuss with Bill

Reply:
was asked to speak to Tobias about it (will post here on findings and in the Spartacus tribe)

Reply:
Correct, this is considered a _breaking change_. When an existing customer has used the component, and applied style rules using the `row` class, it will no longer work for the customer, as we're dropping the class. 

The new architectural direction regarding bootstrap / flexbox should only be applied to new stuff, as we can't change existing css implementations before doing a major release.

Reply:
fixed